api/src/main/java/org/pmiops/workbench/genomics/GenomicExtractionService.java

```diff
@@ -129,7 +129,7 @@ public GenomicExtractionService(
                               dbSubmission.getSubmissionId());
 
                   TerraJobStatus status =
-                      genomicExtractionMapper.convertJobStatus(firecloudSubmission.getStatus());
+                      genomicExtractionMapper.convertJobStatus(firecloudSubmission);
```

Comment:
Not a part of this PR's changes but I noticed this in Terra UI's code while looking around.

https://github.com/DataBiosphere/terra-ui/blob/fa71713c7633485839121538960fd81cc6f376ed/src/pages/workspaces/workspace/JobHistory.js#L37

We could follow the same logic in L119 if we want to be consistent with them since the submission status is already being cached.

Reply:
oh ok. If we're caching our computed job status then I think its fine to leave it as is.